sklearn/discriminant_analysis.py

```diff
@@ -181,6 +285,22 @@ class LinearDiscriminantAnalysis(BaseEstimator, LinearClassifierMixin,
 
         .. versionadded:: 0.17
 
+    covariance_estimator : estimator, default=None
+        The covariance estimator is a BaseEstimator with a fit
+        method and a ``covariance_`` attribute like the estimators in
+        `sklearn.covariance`
+
+        if None the shrinkage parameter drives the estimate.
```

Comment:
Is the shrinkage ignored if we pass a `covariance_estimator`? If so it should be indicated in the docstring of shrinkage, and we should error if both are passed.